### DISCUSSION TITLE
auto-sync based on file modif

### DISCUSSION THREAD

**Discussion by imed-ghomari** [remotely-save/remotely-save#47] [Ideas]
Hi, another option to auto-sync, instead of syncing in time intervals, only sync when changes such as file creation, file deletion, file modification are detected, to save battery.
Also, sync only when a certain amount of time has passed since last modif, to avoid excessive syncing.

**Comment by fyears** [comment]
> Also, sync only when a certain amount of time has passed since last modif, to avoid excessive syncing.

What if something changed on the remote? How could the plugin “automatically” know that?

**Reply by imed-ghomari** [reply]
Good question, I haven't thought of that. I guess check the remote on time intervals, but check the local files only on modif. Would that work ?

**Comment by fyears** [comment]
local auto sync has been implemented
